packages/web-twig/tests/Twig/PropsExtensionTest.php

```diff
@@ -360,6 +360,34 @@ public function useStylePropDataProvider(): array
                 'className' => null,
                 'style' => 'position: absolute;',
             ]],
+            'simple spacing style prop' => [[
+                'margin' => 'space-100',
+            ], [
+                'className' => 'm-100',
+                'style' => null,
+            ]],
+            'complex spacing style prop' => [[
+                'marginX' => [
+                    'mobile' => 'space-100',
+                    'tablet' => 'auto',
+                    'desktop' => 'space-200',
+                ],
+            ], [
```

Comment:
I'd like to see a mixed example too: with an object, but not all breakpoints.

Reply:
Cool. BTW, #1235 is exactly the reason for what I mean here (I didn't know about the PR at the time of writing this!).

Reply:
Which brings me to my prior question on mixing style prop with unsafe style: could we have a test for that too? 🙏🏻

Reply:
I added tests with combination with unsafe style and classname. I tried to add test with prefix, but I wasn't able to make it work in the test environment.